docs/automated-process.md

```diff
@@ -11,6 +11,34 @@ release process. The automated processes uses GitHub Actions to automate the
     `${PLACEHOLDER_VALUE}` where `PLACEHOLDER_VALUE` describes the value to
     specify.
 
+## Setup for the release
+
+### Credentials
+
+- Access to https://github.com/ansible-community/ansible-build-data repo
+- Access to https://github.com/ansible/ansible repo
```

Comment:
What do you mean by access? Nothing needs to be committed to ansible/ansible as part of the community release process.